noxfile.py

```diff
@@ -179,9 +176,15 @@ def unit_tests(session: nox.Session) -> None:
         "polyfactory",
     )
 
+    paths = session.posargs or ["tests/unit_tests"]
+
+    # Launch is not supported on 3.8
+    if session.python == "3.8":
```

Comment:
What is the minimal version of python that launch supports?

Reply:
3.9

Reply:
Ok so this is fine, just assumed we could future proof it